src/CoreWCF.RabbitMQ/src/CoreWCF/Channels/RabbitmqTransportPump.cs

```diff
@@ -17,59 +20,117 @@ namespace CoreWCF.Channels
 {
     public class RabbitMqTransportPump : QueueTransportPump
     {
```

Comment:
This isn't a type that would need to be instantiated by developer code. I think this should be internal.